src/sizes.rs

```diff
@@ -806,9 +806,16 @@ mod extra_sizes {
     pub type U4080 = uint!(0 0 0 0 1 1 1 1 1 1 1 1);
 
     // ML-DSA sizes
+    //
+    // Includes the public key, private key, and signature sizes not covered elsewhere, as well as
+    // some intermediate value sizes.
     pub type U2420 = uint!(0 0 1 0 1 1 1 0 1 0 0 1);
-    pub type U3293 = uint!(1 0 1 1 1 0 1 1 0 0 1 1);
```

Comment:
We've already published a crate with this size, which would make this a breaking change.

You can leave a TODO comment to remove it in a future version (or ideally we find a solution to #81 which allows 3rd party size definitions)

Reply:
Updated to re-add this value, and added a comment explaining that it is not necessary for ML-DSA.